server/schema/archiveSchema.js

```diff
@@ -0,0 +1,27 @@
+module.exports = function (knex) {
+  return knex.schema.createTableIfNotExists('archive', function(table) {
+    table.increments('archive_id').primary();
+    table.integer('stock_id')
+          .references('s_id')
+          .inTable('stocks');
+    table.string('symbol');
+    table.string('bid');
```

Comment:
Need to check the data types for all the stock related data coming in from Yahoo